src/service/balance-service.ts

```diff
@@ -305,12 +309,12 @@ export default class BalanceService {
     if (skip) recordsQuery += `offset ${skip} `;
 
     const balances = await connection.query(recordsQuery, parameters);
-
     if (balances.length > 0 && balances[0].amount === undefined) {
       throw new Error('No balance returned');
     }
+    let count = (await connection.query(query, parameters)).length;
```

Comment:
Waarom is deze `const` nu een `let` geworden? Hij lijkt namelijk nergens anders geredefined te worden?